mct_quantizers/__init__.py

```diff
@@ -13,22 +13,26 @@
 # limitations under the License.
 # ==============================================================================
 
+
 __version__ = "1.2.0"
 
 
 from mct_quantizers.common.base_inferable_quantizer import QuantizationTarget, BaseInferableQuantizer, mark_quantizer
 from mct_quantizers.common.quant_info import QuantizationMethod
-from mct_quantizers.keras.activation_quantization_holder import KerasActivationQuantizationHolder
-from mct_quantizers.pytorch.activation_quantization_holder import PytorchActivationQuantizationHolder
-from mct_quantizers.keras.load_model import keras_load_quantized_model
-from mct_quantizers.keras.quantize_wrapper import KerasQuantizationWrapper
-from mct_quantizers.pytorch.load_model import pytorch_load_quantized_model
-from mct_quantizers.pytorch.quantize_wrapper import PytorchQuantizationWrapper
 
 from mct_quantizers.common import constants
-from mct_quantizers.keras import quantizers as keras_quantizers
-from mct_quantizers.pytorch import quantizers as pytorch_quantizers
 
-from mct_quantizers.pytorch.onnxruntime_session_options import get_ort_session_options
+if constants.FOUND_TF:
```

Comment:
All these API functions/classes have protection against missing tf/torch packages in the function/class level (with FOUND_TF or FOUND_TORCH).
So you don't need to add it here as well.
Did you encounter any missing FOUND_TF/FOUND_TORCH in any of the APIs? If so, it would be better to add it there.

Reply:
I don't use Tensorflow/Keras and importing model_optimization_toolkit would crask there without guarding keras-specific or pytorch-specific imports.

Reply:
But it has protection in each of the relevant files. You can see for example [here](https://github.com/sony/mct_quantizers/blob/6569c1b697909b49c8f3069af34ea9d05267cf54/mct_quantizers/keras/quantizers/base_keras_inferable_quantizer.py#L20) and [here](https://github.com/sony/mct_quantizers/blob/6569c1b697909b49c8f3069af34ea9d05267cf54/mct_quantizers/keras/quantize_wrapper.py#L23) we use `if FOUND_TF` (same in pytorch).
If you think that it is better to gather them all under a single condition in the init file that's sounds good, but please remove the duplicated conditions from the other files and add a dedicated error message in case the required framework not found.